themes/cypress/source/css/_partial/toc.scss

```diff
@@ -79,10 +79,6 @@
   list-style: none;
 }
 
-.toc-item.toc-level-3 {
-  display: none;
-}
-
```

Comment:
Because Hexo is now generating a TOC with a maximum depth of `2`, there aren't even any TOC item links with a level of 3 or higher. As such we no longer need to hide them because they're just not there in the generated markup.